lib/puppet-lint/tasks/release_test.rb

```diff
@@ -86,6 +86,9 @@ def with_puppet_lint_head
     'puppetlabs/puppetlabs-chocolatey',
     'voxpupuli/puppet-archive',
     'voxpupuli/puppet-collectd',
+    'voxpupuli/puppet-prometheus',
+    'voxpupuli/puppet-borg',
+    'voxpupuli/puppet-nginx',
     'garethr/garethr-docker',
     'sensu/sensu-puppet',
     'jenkinsci/puppet-jenkins',
```

Comment:
Wasn't this one transferred and is now `voxpupuli/jenkins`?

Reply:
yes. updated it